sig-cluster-lifecycle/annual-report-2022.md

```diff
@@ -4,64 +4,93 @@
+   - We started a Cluster API release team modeled from the Kubernetes release team, and this allowed the introduction of a quarterly release cadence for the entire 2023.
+  
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
-
-
+   - SIG Cluster Lifecyle has not added any new KEPs that affect the whole group or the wider Kubernetes.
+   New KEPs and KEP updates were only done for [kubeadm](https://git.k8s.io/enhancements/keps/sig-cluster-lifecycle/kubeadm),
+   which is the only subproject that is part of the Kubernetes release artifacts.
+   Most of the activity in this group happens outside of the KEP process.
+   Individual projects have implemented their own proposal tracking means, such
+   as the [Cluster API CAEP process](https://sigs.k8s.io/cluster-api/docs/proposals).
+   - A general theme across subprojects is driving them to maturity.
+   With the exception of projects like kubeadm, minikube, kOps and kubespray most of
+   our other subprojects are fairly new in Kubernetes years.
 
 3. KEP work in 2022 (v1.24, v1.25, v1.26):
 
+As noted above, only KEP work for kubeadm was done in 2021.
+Some KEPs were retroactively updated to GA, since the work there was done without KEP updates.
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - ["kubernetes-sigs/cluster-api-addon-provider-helm"](https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/blob/main/OWNERS)
+   - [etcdadm](https://github.com/kubernetes-sigs/etcdadm)
+   - [cluster-addons](https://github.com/kubernetes-sigs/cluster-addons)
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - We started collecting metrics from all of our subprojects using an
+     [yearly survey](https://forms.gle/xZn8DXww4XxPsXvCA). It contains a number of questions
+     related to project health, OWNERS files, contributor onboarding, etc. What we saw for this annual
+     report is that our OWNERS files and SIG README.md are mostly up-to-date and that subprojects are doing
+     what they can to onboard new contributors. We are considering preparing some actions to better educate
+     our subprojects about onboarding contributors and graduating more OWNERS. We did not do a survey
+     for 2022.
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - As per the 2021 survey, we are seeing a good number of projects to keep their contributing
+     documentation up-to-date. We are drafting action items for the leads to improve the understanding
+     around [CONTRIBUTING.md] management. We have a good number of projects that have participated in LFX and GSoC.
```

Comment:
> a good number of projects

how many?

Reply:
@neolit123 do you have some data?

Reply:
this is copy paste from last year. IIRC, k0ps, cluster-addons, kubeadm (no student applied), @justinsb might remember better 

language can be from "good number" to "a few"